src/pyasm/search/upgrade/project/sthpw_upgrade.py

```diff
@@ -21,8 +21,11 @@ class SthpwUpgrade(BaseUpgrade):
     #
     # 4.2.0.a01
     #
-
-
+    def upgrade_v4_2_0_a01_012(my):
+        my.run_Sql(''' 
```

Comment:
it's lowercase run_sql(
I suggest running python upgrade_db.py -f 
before submitting